### PR TITLE
Bug 1989339: Remove log symlinks when executing wipe

### PR DESCRIPTION
**Pull request by harche**
Signed-off-by: Harshal Patil <harpatil@redhat.com>

#### What type of PR is this?

/kind bug

#### Which issue(s) this PR fixes:

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1989339

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cmd/crio/wipe.go [hunk 1]
```diff
@@ -195,7 +201,36 @@ func (c ContainerStore) deleteContainer(id string) {
...
+
+func (c ContainerStore) removeLogSymlinks(id string) error {
+	files, err := ioutil.ReadDir(kubeletLogDir)
+	if err != nil {
+		return err
+	}
+
+	re := regexp.MustCompile(`\S.+-(\S{64})\.log`)
+
+	for _, f := range files {
```
Comment (saschagrunert) [inline cmd/crio/wipe.go:219]: I'm wondering why we have to iterate at all. Can we just construct the logfile name from the ID and then try to remove it?

**Review by saschagrunert — COMMENTED** [review]
Closing the loop: I think we may handle this in the kubelet, since there seems to be a cleanup routine for that already: https://github.com/kubernetes/kubernetes/blob/1a845ccd076bbf1b03420fe694c85a5cd3bd6bed/pkg/kubelet/kuberuntime/kuberuntime_gc.go#L351

File: cmd/crio/wipe.go (hunk 1 above)
Comment (harche) [reply cmd/crio/wipe.go:219]: Makes sense.

**Pull request closed without merging**